runtime/layers/function/Dockerfile

```diff
@@ -6,9 +6,11 @@ WORKDIR /opt
 COPY bootstrap bootstrap
 COPY php.ini bref/etc/php/conf.d/bref.ini
 
+# Remove PHP-FPM
+RUN rm bref/sbin/php-fpm bin/php-fpm
+
+# Build the final image from the lambci image that is close to the production environment
 FROM lambci/lambda:provided
 
-WORKDIR /
```

Comment:
We override it just below.

Reply:
C'est un reste d'une epoque où le copy n'etait pas en absolu  :/